plugins/tungsten-event-touch/index.js

```diff
@@ -157,14 +155,7 @@ if (typeof document.createEvent === 'function') {
   setListener(touchevents.touchend + (justTouchEvents ? '' : ' mouseup'), onTouchEnd);
   setListener(touchevents.touchmove + (justTouchEvents ? '' : ' mousemove'), onTouchMove);
 
-  module.exports = {
-    events: [
-      'swiperight',
-      'swipeleft',
-      'swipedown',
-      'swipeup',
-      'dbltap',
-      'tap'
-    ]
-  };
 }
+
+// Export an empty function to conform to the plugin API
+module.exports = function() {};
```

Comment:
@andrewrota Can you foresee any side effects we might encounter by returning an empty function here and registering it with `addEventPlugin`?  This keep the api uniform.